tools/monitoring/log_parser/database.go

```diff
@@ -32,6 +32,23 @@ const (
 		) VALUES (?,?,?,?,?,?)`
 )
 
+// ErrorLog stores a row in the "ErrorLogs" db table
+// Table schema: github.com/knative/test-infra/tools/monitoring/mysql/schema.sql
+type ErrorLog struct {
+	Pattern     string
+	Msg         string
+	JobName     string
+	PRNumber    int
+	BuildLogURL string
+	TimeStamp   time.Time
+}
+
+// Str returns the string representation of the struct used in alert message
+func (e ErrorLog) Str() string {
```

Comment:
Call this `String()` so it implements the Stringer interface. https://tour.golang.org/methods/17